web/src/components/auth/InputGroup.jsx

```diff
@@ -0,0 +1,58 @@
+import { useState } from "react";
+
+
+export default function InputGroup(props) {
+  // props
+  const { type, contenttitle } = props;
+
+  // placeholder for the input
+  const placeHolder = `Enter your ${contenttitle}`;
+
+  // state to toggle the input type
+  const [show, setShow] = useState(false);
+
+  const handleClick = () => {
+    setShow(!show);
+  };
+
+  if (type === "password") {
+    return (
+      <div {...props} className="input-group">
+        <label htmlFor={contenttitle} className="input-label">
+          {contenttitle}
+        </label>
+        <div className="relative">
+          <input
+            type={show ? "text" : type}
+            className="form-control"
+            id={contenttitle}
```

Comment:
camel case to contentTitle

Reply:
it's giving error in console

Reply:
> The pages that require form fields, wrap them inside a form tag, add an onSubmit event and prevent default form action

ok so this, will be done in the auth page not the way you wrote it, I'll fix it now sha
thanks